docs/container.md

```diff
@@ -99,28 +102,23 @@ To disable this set the environment variable IMAGES to False.
 Controls whether videos with predicted bounding boxes are created in the output directory for input videos.
 See the `FPS` and `EVERY` option for even more control over the output video that gets created.
 
-If `REBUILD` is False then one output image is created for each frame of the video. The output image names will be the same
-name as the video file, with a `_1`, `_2`, etc appended. 
-So an input video named `FrontDoor-Oct-14-2020.mp4` will cause output images named `FrontDoor-Oct-14-2020_1.jpg`,
-`FrontDoor-Oct-14-2020_2.jpg`, etc to be created in the output directory.
```

Comment:
When REBUILD is false, videos are simply not rebuilt. The inference on frames is separate to this and is controlled by the IMAGES option. This allows the user to have both videos and individual frames without having to rerun the container.

Reply:
The desired default behavior here is that input images -> output images and input videos -> output videos.  We don't want videos producing frames and video both by default.